### ISSUE TITLE
Be able to aggregate by or stack by attribute/custom field in portfolio items

### ISSUE THREAD

**Issue by luklau88** [RallyCommunity/CustomChart#22]
Hi Kyle. Thank you so much for this app. This is one of my most used App. 

It will be great and I wonder if it can aggregate by or stack by attribute/custom field in portfolio items when the chart is displaying at the User Story level. Thanks! Luk

**Comment by krmorse** [comment]
Cool, glad to hear you find it useful!  So you basically want the ability to traverse attributes...  Say, a chart of stories aggregated by Feature.my_customField?  Is that right?  Can you provide a more concrete example of what you're proposing?

It also may be tough to integrate this into the general purpose custom chart app, but I'll withhold judgment until I hear from you again.

**Comment by luklau88** [comment]
Yes, exactly, "a chart of stories aggregated or stacked by Feature.my_customField". For example, if I have some sort of custom field indicate category or value at the porfolio level, with this feature, I can see a chart that shows all the user stories in categories across portfolio items...

**Comment by michiQ** [comment]
My users would <3 this too.
